### DISCUSSION TITLE
Do you have any plans on merging ibhagwan's pull request?

### DISCUSSION THREAD

**Discussion by prsh11** [yshui/picom#721] [Q&A] [answered]
It's been a year since ibhagwan started the pull request. It has been abandoned since then. Everyone of us want rounded and dual_kawase for your pc.

**Comment by yshui** [comment, marked as answer]
Rounded corner and kawase are already merged

**Reply by prsh11** [reply]
so how do I use it?

using `picom --experimental-backends`?

**Reply by prsh11** [reply]
also a sample config?

**Reply by tryone144** [reply]
After #716 (or #658 for complete support) has been merged you can use rounded corners and dual-kawase with:
```console
$ picom --experimental-backends --backend glx --blur-method dual_kawase --blur-strength 7 --corner-radius 24
```
or by adding these options to your config file (except for `--experimental-backends`).

**Reply by prsh11** [reply]
added this but no effect

**Reply by prsh11** [reply]
can you give the lines to add to config for blur and rounded corners?

**Reply by godalming123** [reply]
here is some code:
```
#blur rule
blur: {
    method = "dual_kawase";
    strength = 10;
};

blur-background-exclude = [
  "window_type = 'dock'",
  #"window_type = 'desktop'",
  #"class_g = 'URxvt'",
  "class_g = 'dwm'",
  "class_g = 'plank'",
  #
  # prevents picom from blurring the background
  # when taking selection screenshot with `main`
  # https://github.com/naelstrof/maim/issues/130
  "class_g = 'slop'",
  "_GTK_FRAME_EXTENTS@:c"
];

#rounded corners
corner-radius = 8;
rounded-corners-exclude = [
    "class_g = 'polybar'",
    "window_type = 'dock'",
    "class_g = 'tint2'",
];
round-borders = 1;
detect-rounded-corners = true;
```
Now all you need to do is run:
```
picom --experimental-backends
```

**Reply by godalming123** [reply]
Also you may need to download the picom-git version from the aur or build from source as I'm not sure if this has been added to the official repos

**Reply by prsh11** [reply]
oh ok thx

**Reply by FelixBrakel** [reply]
For me the blur still acts as if the corners were not rounded as described in #775 with this config, also what are `round-borders`? I can't find it anywhere in the man page.

**Reply by godalming123** [reply]
In ibhagwans fork you can use that setting to round the inside borders as well as the outside.

**Reply by yshui** [reply]
@FelixBrakel @godalming123 the `round-borders` feature is always on when corners are rounded. since otherwise things just look wrong.

**Reply by FelixBrakel** [reply]
Ah, ok that makes sense.
